var/spack/repos/builtin/packages/cantera/package.py

```diff
@@ -46,112 +53,150 @@ class Cantera(Package):
     variant('matlab',     default=False,
             description='Build the Cantera Matlab toolbox')
 
-    # Required dependencies
-    depends_on('scons', type='build')
-
     # Recommended dependencies
+    depends_on('eigen', when='@2.3.0:+eigen', type='build')
+    depends_on('fmt',   when='@2.3.0:+fmt')
+    depends_on('googletest', when='@2.3.0:+googletest')
```

Comment:
Turns out these 3 dependencies are hard requirements. You can either install them externally, or you can clone the git repos into the cantera source tree and cantera will build them for you.